src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,21 @@
+.stars {
+  display: flex;
+  align-items: center;
+
+  list-style-type: none;
+
+  &__star {
+    height: 16px;
+    width: 16px;
+
+    background: url(/src/images/star.svg) no-repeat center;
+  }
+
+  &__star:not(:last-child) {
+    margin-right: 4px;
+  }
+
+  &--4 &__star:nth-child(-n + 4) {
+    background: url(/src/images/star-active.svg) no-repeat center;
+  }
```

Comment:
It should be the same functionality as in the stars task, if the modifier is `stars--3` you should show 3 active stars and so on